### DISCUSSION TITLE
Where are backups configured?

### DISCUSSION THREAD

**Discussion by enboig** [azukaar/Cosmos-Server#350] [Q&A]
I read there is a way to backup apps (I would not trust my data to an app if it cannot make backups); but I couldn't find them in menus. How are they configured?

**Comment by azukaar** [comment]
The backup feature for apps is not in Cosmos yet, planned for 0.18

**Reply by enboig** [reply]
Thanks, I will continue checking other aspects of Cosmos.

By the way, if Cosmos could make use of some tool like restic/rustic to allow deduplication of backups, and store them offsite encrypted, it would be a plus (tags usage could allow making relationships between apps and their backup)
